src/TransferOption.js

```diff
@@ -1,46 +1,69 @@
-import React from 'react'
+import React, { useRef } from 'react'
 import cx from 'classnames'
 import propTypes from '@dhis2/prop-types'
 
 import { colors } from './theme.js'
 
+const DOUBLE_CLICK_MAX_DELAY = 250
```

Comment:
Could we adjust this to `500`? I've read around and I see that the [default value in Windows is 500](https://docs.microsoft.com/en-gb/windows/win32/api/winuser/nf-winuser-setdoubleclicktime?redirectedfrom=MSDN), so I think that's a good place to start. I've tested `500` and it seems reasonable.

In an ideal world we'd use the users OS setting as double-click-speed is a common setting to adjust, but I'm not sure the browser has access to that.